config/service_alias.go

```diff
@@ -22,9 +22,49 @@ const (
 // gateway.QoSService interface.
 // THis is to handle requests for a given service ID.
 var ServiceQoSTypes = map[protocol.ServiceID]ServiceQoSType{
-	// TODO_IMPROVE(@commoddity): Add all EVM service IDs here.
+	// All Morse EVM Services as of 12/17/2024
```

Comment:
```suggestion
	// All Morse EVM Services as of 12/17/2024 (#103).
	// TODO_TECHDEBT(@fredteumer): Revisit and consider removing these once #105 is complete.
```

Reply:
The reason it's 12/18/2024 is because there are other services today but they will be deprecated tomorrow. I want to use the accurate date. Will add the reference to #103 and other suggestions shortly